tests/helpers/package-cache.js

```diff
@@ -429,26 +508,22 @@ PackageCache.prototype = {
    * @param {String} label The label for the cache.
    * @param {String} type The type of package cache.
    * @param {String} manifest The contents of the manifest file for the cache.
+   * @param {Array} links Packages to elide for install and link.
    * @return {String} The directory on disk which contains the cache.
    */
-  create: function(label, type, manifest) {
+  create: function(label, type, manifest, links) {
     type = checkDowngrade(type);
-
-    // Set up the default Ember CLI link.
-    // Easier to do this every time than try and lock it out.
-    if (!isEmberCLILinked[type] && this.options.linkEmberCLI === true && (type === 'npm' || type === 'yarn')) {
-      commands[type].invoke('link', { cwd: emberCLIPath });
-      isEmberCLILinked[type] = true;
-    }
+    links = links || [];
 
     // Save metadata about the PackageCache invocation in the manifest.
-    var packageManagerVersion = require(type + '/package.json').version;
+    var packageManagerVersion = commands[type].invoke('--version').stdout;
```

Comment:
I elected not to run it through my `translate` lookup since it is consistent across all three package managers currently supported. It seemed like an unnecessary layer of indirection. If we ever add a package manager that needs something different this will get updated.

Going with "YAGNI" here.